stablehlo/integrations/python/setup.py

```diff
@@ -0,0 +1,45 @@
+"""This setup.py builds a wheel file assuming that StableHLO is already built
```

Comment:
Not the best setup.py but it gets the job done.
I link to some other examples like https://github.com/makslevental/pristine-llvm-release which does a lot more in the setup.py by modeling the CMake build.

The upside is that it removes some of the hard-coded aspects above; the downside is the setup.py is a lot more complicated.

I'm not the best at Python so maybe we can ask someone from OpenXLA to review this after landing it; but it seems to work for now especially for nightly builds.